stdlib/gc.mli

```diff
@@ -241,21 +241,18 @@ type control =
 
 external stat : unit -> stat = "caml_gc_stat"
 (** Return the current values of the memory management counters in a
-   [stat] record that represent the program's total memory stats.
-   This function causes a full major collection. *)
+    [stat] record that represent the program's total memory stats. Due to
+    per-domain buffers it may only represent the state of the program's
+    total memory usage since the last minor collection.
+    In order to get exact values, you have to call [full_major]
+    immediately before [stat]. *)
```

Comment:
```suggestion
    {!type:stat]}record that represents the program's total memory stats. Due to
    per-domain buffers it may only represent the state of the program's
    total memory usage since the last minor collection.
    To get exact values, call {!full_major}
    immediately before {!stat}. *)
```